showcase/src/assets/examples/table/with-custom-styling.scss

```diff
@@ -0,0 +1,19 @@
+.even td {
+  background-color: #d1e1ff;
```

Comment:
Regarding all the styles in this SCSS file, I suggest we remove the element selectors like `td` and `th`... otherwise it feels like we are cheating setting directly the styles for specific HTML tags 😆 which might not always be the case if we change our Table component at some point